.circleci/config.yml

```diff
@@ -15,6 +21,10 @@ jobs:
       - run:
           name: Installing...
           command: node common/scripts/install-run-rush.js install
+      - save_cache:
```

Comment:
Does this happen regardless of whether or not the shrinkwrap.yaml has changed?

Reply:
My understanding is that it saves every time.

Reply:
Do we care/can we avoid that when there's no update?

Reply:
Otherwise lgtm

Reply:
Fair question, if it net doesn't improves build times I'll revert. Not sure if that type of logic can be added or it may already be implicitly happening if there's a key match on save.